scripts/src/client.ts

```diff
@@ -122,7 +122,7 @@ class PaymentStream {
 	}
 }
 
-class Wallet implements KinWallet {
+export class Wallet implements KinWallet {
```

Comment:
Why do you need to export the `Wallet` class? I thought it's supposed to be a hidden implementation.

Reply:
you should add `burn` to the KinWallet interface to make it accessible instead of exporting the implementation class